fake_ubersmith/main.py

```diff
@@ -21,24 +21,24 @@
 from fake_ubersmith.api.methods.uber import Uber
 from fake_ubersmith.api.ubersmith import UbersmithBase
 
-# TODO (wajdi) Make configurable passed parameter
-port = 9131
 
-app = Flask('fake_ubersmith')
+def run():
+    # TODO (wajdi) Make configurable passed parameter
```

Comment:
Yeah. Just a reminder that this should get done eventually to allow for different ports, but not super important right now. Future me is forgetful.